tests/phpunit/tests/rest-api/rest-users-controller.php

```diff
@@ -856,10 +856,83 @@ public function test_get_items_invalid_roles() {
 		$request->set_param( 'roles', 'steakisgood' );
 		$response = rest_get_server()->dispatch( $request );
 		$data     = $response->get_data();
-		$this->assertCount( 0, $data );
-		$this->assertSame( array(), $data );
+		$this->assertIsArray( $data );
+		$this->assertEmpty( $data );
+	}
+
+	public function test_get_items_capabilities() {
+		wp_set_current_user( self::$user );
+
+		$tango = $this->factory->user->create(
+			array(
+				'display_name' => 'tango',
+				'role'         => 'subscriber',
+			)
+		);
+		$yolo  = $this->factory->user->create(
+			array(
+				'display_name' => 'yolo',
+				'role'         => 'author',
+			)
+		);
```

Comment:
Can `self::$author2` be created in `wpSetUpBeforeClass()` instead of creating a second author both here and in `test_get_items_invalid_capabilities()`

Same for subscriber role if another test is already using a second subscriber.

Reply:
Good point regarding reusing existing users where possible 👍